rails/config/routes.rb

```diff
@@ -6,6 +6,7 @@
   # root 'welcome#index'
   namespace :api do
     resources :posts
+    resources :analytics
```

Comment:
- [ ] TODO: Yes. I like tidy rake routes. (even though i'll never run it in this app. Ember FTW)